src/winmain.cpp

```diff
@@ -217,17 +217,35 @@ static size_t ratio = 0;
 
 static size_t setProgress(HWND, double t, double d, double, double)
 {
-	while (stopDL)
-		::Sleep(1000);
-	size_t step = size_t(d * 100.0 / t - ratio);
-	ratio = size_t(d * 100.0 / t);
+	// Once downloading is finish (100%) close the progress bar dialog
+	// as there is no need to keep it opened because
+	// new dailog asking to close the app will appear (if specified classname in configuration)
+	static bool IsDialogClosed = false;

```

Comment:
Please use `isDialogClosed` instead of `IsDialogClosed`.

Reply:
Done!